src/persist-client/src/cli/bench.rs

```diff
@@ -76,8 +76,8 @@ async fn bench_s3(args: &S3FetchArgs) -> Result<(), anyhow::Error> {
         let mut fetches = Vec::new();
         for part in snap.iter().flat_map(|x| x.parts.iter()) {
             let key = match part {
-                BatchPart::Hollow(x) => x.key.complete(&shard_id),
-                BatchPart::Inline { .. } => continue,
+                RunPart::Single(BatchPart::Hollow(x)) => x.key.complete(&shard_id),
+                _ => continue,
```

Comment:
Sure! I will replace this with just a fetch for the hollow run for now, since doing a recursive fetch seems to go against the spirit of this benchmark... but let me know if you prefer otherwise.

Reply:
It's meant to be recursive! Totally fine to move the flattening one top in the setup, though, and just have a list of blob keys to fetch here

Reply:
Aight, done!